src/T2DMap.cpp

```diff
@@ -2800,28 +2800,25 @@ void T2DMap::mousePressEvent(QMouseEvent* event)
 
             if (!mMapViewOnly) {
                 if (selectionSize == 0) {
-                    mpCreateRoomAction = new QAction(tr("Create room", "Menu option to create a new room in the mapper"), this);
-                    mpCreateRoomAction->setToolTip(tr("Create a new room here"));
+                    mpCreateRoomAction = new QAction(tr("Create new room here", "Menu option to create a new room in the mapper"), this);
                     connect(mpCreateRoomAction.data(), &QAction::triggered, this, &T2DMap::slot_createRoom);
                     popup->addAction(mpCreateRoomAction);
                 }
 
                 if (selectionSize > 0) {
-                    auto moveRoom = new QAction(tr("Move", "2D Mapper context menu (room) item"), this);
-                    moveRoom->setToolTip(tr("Move room", "2D Mapper context menu (room) item tooltip"));
+                    auto moveRoom = new QAction(tr("Move room(s)", "2D Mapper context menu (room) item", selectionSize), this);
                     connect(moveRoom, &QAction::triggered, this, &T2DMap::slot_moveRoom);
                     popup->addAction(moveRoom);
                 }
 
                 if (selectionSize > 0) {
-                    auto roomExits = new QAction(tr("Exits", "2D Mapper context menu (room) item"), this);
-                    roomExits->setToolTip(tr("Set room exits", "2D Mapper context menu (room) item tooltip"));
+                    auto roomExits = new QAction(tr("Set room exits...", "2D Mapper context menu (room) item"), this);
                     connect(roomExits, &QAction::triggered, this, &T2DMap::slot_setExits);
                     popup->addAction(roomExits);
                 }
 
                 if (selectionSize == 1) {
-                    auto customExitLine = new QAction(tr("Custom exit line", "2D Mapper context menu (room) item"), this);
+                    auto customExitLine = new QAction(tr("Customize exit line...", "2D Mapper context menu (room) item"), this);
```

Comment:
This only allows the *Addition* of a custom exit line - so perhaps we should stay that instead:
```suggestion
                    auto customExitLine = new QAction(tr("Add custom exit line...", "2D Mapper context menu (room) item"), this);
```

Reply:
In a different discussion @vadi2 hinted the word "custom" was superfluous, that's why I dropped it here sort of, including it in the verb instead.
I would settle for "Create exit line" in harmony with the actions called "Create room" and "Create label" as well. Or "Create custom exit line" if that word must stay.